src/components/questions-accordion/QuestionsAccordion.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+
+import ArrowForwardIosSharpIcon from '@mui/icons-material/ArrowForwardIosSharp'
+
+import { AccordionItem, VariantEnum } from '~/types'
+import { styles } from '~/components/questions-accordion/QuestionsAccordion.styles'
+import Accordions from '../accordion/Accordions'
+
+interface QuestionsAccordion {
+  items: AccordionItem[]
+  title?: string
+}
+
+const QuestionsAccordion: FC<QuestionsAccordion> = ({ items, title = '' }) => {

```

Comment:
also this component if it is reusable should provide props to acordions such as icon,sx,etc

Reply:
i think that this component is special only for offer details page and it to make this component reusable it will be needed one more wrapper

Reply:
if it is only for offer details it should be container not component, but I shink we will need such component in future, do it as component with multiple open accordions and use it for your needs on offer details